packages/core/jsx-runtime/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@theme-ui/core-jsx-runtime",
```

Comment:
The bundles for jsx-runtime and jsx-dev-runtime need to be accessible via the paths `foo/jsx-runtime` and `foo/jsx-dev-runtime` where `foo` can be any path but the rest is fixed.

I added them both to `@theme-ui/core` so one can use the comment

```js
/** @jsxImportSource @theme-ui/core */
const foo = <div sx={{ color: 'hotpink' }}>Hello <b>world</b></div>
```

which is then transformed to

```js
import { jsx as _jsx } from '@theme-ui/core/jsx-runtime'
import { jsxs as _jsxs } from '@theme-ui/core/jsx-runtime'

const foo = _jsxs('div', {
  sx: {
    color: 'hotpink'
  },
  children: [
    'Hello ',
    _jsx('b', {
      children: 'world',
    }),
  ],
})
```

without the need to install a new dependency.

But maybe it makes sense to expose the same entry points in the `theme-ui` package?

Reply:
I added the JSX runtime to the `theme-ui` package as well in 085a1479. Now both of these comments can be used:

```js
/** @jsxImportSource theme-ui */
/** @jsxImportSource @theme-ui/core */
```

similar to how it worked with the classic JSX runtime.